scripts/sync-pkgs-wireit.js

```diff
@@ -64,10 +64,10 @@ const updateRootPkg = async ({ allPkgs }) => {
   rootPkg.wireit = rootPkg.wireit || {};
   [
     { rootTask: "build", pkgTask: "build" },
+    { rootTask: "build:lib:esm", pkgTask: "build:lib:esm" },
```

Comment:
I like how the script names match; maybe we should do this for the rest of the root scripts?  (remove the `:pkgs` part)

Reply:
The issue is for some of our tasks like lint and others, we also have to add a root task for things outside of the packages. Build has no root tasks